sites/all/themes/ocha_basic/img/icons/sprite.symbol.html

```diff
@@ -174,7 +174,7 @@
 		<symbol viewBox="0 0 48 48" id="search" xmlns="http://www.w3.org/2000/svg"><path d="M46.12 41.88L33.66 29.41a18.06 18.06 0 1 0-4.24 4.24l12.46 12.47a3 3 0 0 0 4.24-4.24zM7 19a12 12 0 1 1 12 12A12 12 0 0 1 7 19z"/></symbol>
 		<symbol viewBox="0 0 48 48" id="see" xmlns="http://www.w3.org/2000/svg"><path d="M47.38 21.8C41.92 12.78 33.83 9 24 9S6.08 12.78.62 21.8a4.22 4.22 0 0 0 0 4.39C6.08 35.22 14.17 39 24 39s17.92-3.78 23.38-12.8a4.23 4.23 0 0 0 0-4.4zM24 32a8 8 0 1 1 8-8 8 8 0 0 1-8 8z"/></symbol>
 		<symbol viewBox="0 0 48 48" id="selected" xmlns="http://www.w3.org/2000/svg"><path d="M18.5 39.75a4 4 0 0 1-2.83-1.17l-12.5-12.5a4 4 0 0 1 5.66-5.66l9.67 9.67L39.17 9.42a4 4 0 0 1 5.66 5.66l-23.5 23.5a4 4 0 0 1-2.83 1.17z"/></symbol>
-		<symbol viewBox="0 0 48 48" id="selected--highlighted" xmlns="http://www.w3.org/2000/svg"><path fill="#80CBFF" d="M18.5 39.75a4 4 0 0 1-2.83-1.17l-12.5-12.5a4 4 0 0 1 5.66-5.66l9.67 9.67L39.17 9.42a4 4 0 0 1 5.66 5.66l-23.5 23.5a4 4 0 0 1-2.83 1.17z"/></symbol>
+		<symbol viewBox="0 0 48 48" id="selected--highlighted" xmlns="http://www.w3.org/2000/svg"><path fill="#fff" d="M18.5 39.75a4 4 0 0 1-2.83-1.17l-12.5-12.5a4 4 0 0 1 5.66-5.66l9.67 9.67L39.17 9.42a4 4 0 0 1 5.66 5.66l-23.5 23.5a4 4 0 0 1-2.83 1.17z"/></symbol>
```

Comment:
@rupl this is one example where I need to adjust the fill value of the SVG instead of using CSS fill property and the generic SVG (it's the checkmark for the language switcher, which is set as a background image). It also means I need to rerender the icon sprite. I'm pointing it out to discuss during our next CD meeting.